deployments/quick-start/setup.sh

```diff
@@ -46,14 +47,16 @@ then
     exit 1
 fi
 
+# 2. Setup Vault
+
 # 2.1 Port forward vault
 nohup kubectl port-forward $vaultPod 8200 &
 
-# Set environment variables and authenticate
+# 2.2 Set environment variables and authenticate
 export VAULT_ADDR=http://127.0.0.1:8200
 vault auth vault-root-token
 
-# Set up the Root Certificate Authority
+# 2.2 Set up the Root Certificate Authority
 # Create a Root CA that expires in 10 years: 
```

Comment:
This should be `2.3`. I got the numbering wrong when writing up `quick-start.md`.